js/style/style.js

```diff
@@ -323,6 +323,14 @@ Style.prototype = util.inherit(Evented, {
         return this;
     },
 
+    setLayerZoomRange: function(layerId, minzoom, maxzoom) {
+        this.batch(function(batch) {
+            batch.setLayerZoomRange(layerId, minzoom, maxzoom);
+        });
+
+        return this;
+    },
```

Comment:
I agree you can save a few lines, however, I find the explicit return to be more clear.